IPython/core/interactiveshell.py

```diff
@@ -90,12 +90,17 @@
 try:
     import docrepr.sphinxify as sphx
 
-    def sphinxify(doc):
-        with TemporaryDirectory() as dirname:
-            return {
-                'text/html': sphx.sphinxify(doc, dirname),
-                'text/plain': doc
-            }
+    def sphinxify(oinfo):
+        wrapped_docstring = sphx.wrap_main_docstring(oinfo)
+
+        def sphinxify_docstring(docstring):
+            with TemporaryDirectory() as dirname:
+                return {
+                    'text/html': sphx.sphinxify(wrapped_docstring, dirname),
+                    'text/plain': docstring
```

Comment:
```suggestion
                    'text/plain': docstring,
```

Probably one thing darker is complaining about